code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -38,9 +38,14 @@
 	///Radio connection from the air alarm
 	var/radio_filter_in
 
+	//Enables the use of plunger_act for ending the vent clog random event
+	var/clogged = FALSE
```

Comment:
You can probably use a trait to track this, instead of a var, but the var is fine. Just an idea.